src/utils/update/trust.model.fetch.ts

```diff
@@ -308,7 +308,7 @@ const linkBoToTrust = (beneficialOwner: BeneficialOwnerIndividual | BeneficialOw
   beneficialOwner.trust_ids.push(trust.trust_id);
 };
 
-const mapTrusteeType = (trusteeTypeId: string): RoleWithinTrustType => {
+export const mapTrusteeType = (trusteeTypeId: string): RoleWithinTrustType => {
```

Comment:
Wondering why this function needed the `export` keyword added? Can't see it being newly called from anywhere else in this PR.

Reply:
It doesn't. I exposed the function for rudimentary testing. I thought I had reverted it.

Reply:
export keyword removed.